### PR TITLE
feat: get db status

### PR DESCRIPTION
**Pull request by MutegekiHenry**
### What does this PR do?

- It retrieves the `db status` of a specific database.

### How to test

- Pull branch and test the route `/projects/{project_id}/databases/{database_id}/status`

### CODE REVIEW

File: app/controllers/project_database.py [hunk 1]
```diff
@@ -196,6 +196,42 @@ def get(self, project_id, database_id):
...
+
+        if not database_existant:
+            return dict(
+                status="fail",
+                message=f"Database with id {database_id} not found."
+            ), 404
+
+        # Check the database status on host
+        database_service = DatabaseService()
+        database_connection = database_service.create_db_connection(user=database_existant.user, password=database_existant.password, db_name=database_existant.name)
```
Comment (Mubangizi) [inline app/controllers/project_database.py:220]: Hey @MutegekiHenry You will need to close this connection after checking its success

Comment (Mubangizi) [reply app/controllers/project_database.py:220]: Maybe you can create a function that will close a particular connection

**Review by Mubangizi — CHANGES_REQUESTED** [review]
Great work @MutegekiHenry, just a few recommendations you might need to look at

File: app/controllers/project_database.py [hunk 2]
```diff
@@ -196,6 +196,42 @@ def get(self, project_id, database_id):
         return dict(status='success', data=dict(database=json.loads(database_data))), 200
 
 
+class ProjectDatabaseStatusView(Resource):
```
Comment (Mubangizi) [inline app/controllers/project_database.py:199]: I also do not think we might need an endpoint to return db status.
We might return db status at the endpoint that shows info about a single db. `GET` method on `/databases/<database_id>`
What do you think?

Comment (MutegekiHenry) [reply app/controllers/project_database.py:199]: @Mubangizi you make a good point

File: app/controllers/project_database.py (hunk 1 above)
Comment (MutegekiHenry) [reply app/controllers/project_database.py:220]: sure thing!

**Comment by MutegekiHenry** [comment]
@Mubangizi @inno-asiimwe am closing this so that I wait for the health check tool ALex promised to set up on the db Host that will give us better status details.

**Pull request closed without merging**